src/main/java/core/basesyntax/Buldoser.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Buldoser extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Buldoser started working");
+    }
+
+    public void stopWork() {
+        System.out.println("Buldoser stopped working");
```

Comment:
Similarly, the message in the 'stopWork' method should be more informative. Consider changing it to 'Bulldozer has stopped working.' to clearly indicate that the Bulldozer has stopped its work.